packages/components/src/styles/components/form/super-select.scss

```diff
@@ -0,0 +1,487 @@
+    padding: 4px;
+    padding-right: calc(var(--token-form-control-padding) + 24px); // extra space for the icon
+    padding-left: 11px;
+    color: var(--token-form-control-base-foreground-value-color);
+    line-height: var(--token-typography-body-200-line-height);
+    background-color: var(--token-form-control-base-surface-color-default);
+    background-image: var(--token-form-select-background-image-data-url);
+    background-repeat: no-repeat;
+    background-position: right var(--token-form-select-background-image-position-right-x) center, top 8px left 20px;
+    background-size: var(--token-form-select-background-image-size) var(--token-form-select-background-image-size);
+    border: var(--token-form-control-border-width) solid var(--token-form-control-base-border-color-default);
+    border-radius: var(--token-form-control-border-radius);
+    box-shadow: var(--token-elevation-low-box-shadow);
+
+    &:hover,
+    &.mock-hover {
+      border-color: var(--token-form-control-base-border-color-hover);
+    }
+
+    &:focus,
+    &.mock-focus {
+      border-color: var(--token-color-focus-action-internal);
+      outline: 3px solid var(--token-color-focus-action-external);
+      outline-offset: 0;
+    }
+
+    &[aria-expanded="true"] {
+      &.ember-basic-dropdown-trigger--in-place {
+        border-radius: var(--token-form-control-border-radius);
+      }
+    }
+
+    // Hide default trigger dropdown arrow
+    .ember-power-select-status-icon {
+      display: none;
+    }
+
+    // Trigger content (Single Select):
+    .ember-power-select-placeholder,
+    .ember-power-select-selected-item {
+      flex: 1;
+      margin-left: 0;
+    }
+
+    .ember-power-select-placeholder {
+      color: var(--token-form-control-base-foreground-placeholder-color);
+    }
+
+    &[aria-disabled="true"] {
+      color: var(--token-form-control-disabled-foreground-color);
+      background-color: var(--token-form-control-disabled-surface-color);
+      background-image: var(--token-form-select-background-image-data-url-disabled);
+      border-color: var(--token-form-control-disabled-border-color);
+      box-shadow: none;
+      cursor: not-allowed;
+      opacity: 1;
+
+      .ember-power-select-selected-item {
+        opacity: inherit;
+      }
+    }
+  }
+
+  // DROPDOWN
+  .ember-basic-dropdown-content {
+    max-width: var(--hds-form-super-select-dropdown-max-width);
+    margin: 0;
+    border: var(--token-form-control-border-width) solid var(--token-form-control-base-border-color-default);
+    border-radius: var(--token-form-control-border-radius);
+
+    &.ember-basic-dropdown-content--in-place {
+      box-shadow: var(--token-elevation-high-box-shadow), var(--token-elevation-high-box-shadow);
+    }
+
+    &.ember-basic-dropdown-content--below {
+      margin-top: 4px;
+    }
+
+    &.ember-basic-dropdown-content--above {
+      top: unset !important; // override inline style (TODO: update in JS)
```

Comment:
@KristinLBradley you may have more context than I do on this comment. On another note, the `!important` seems to still be required (I wonder if we can rely on specificity instead)

Reply:
I don't know how we could actually override this in the JS since I don't think it's exposed in the API from what I can tell. Since this is to override an inline style I don't see a reasonable way around using `!important`.

I'll just update the comment for now. Maybe this is something we could address with a PR against ember-power-select down the road.